portfolio/src/main/webapp/script.js

```diff
@@ -50,21 +153,54 @@ function randomMovie() {
   container.innerText = '"' + movie + '"';
 }
 
+
+async function grabUser() {
+  const userInfo = await fetch('/chat-login', {
+                     method: 'POST'
```

Comment:
This should be a GET request because it doesn't change state on the server. https://stackoverflow.com/a/3477374

Reply:
Should I create a new servlet? The "/chat-login" already has a get that does other stuff for user authentication.